### PR TITLE
Review data structures dfs

### PR DESCRIPTION
**Pull request by ateucher**
I think very minor changes to this stuff, at least at a first quick pass. So based on our conversations, I will stop at line 206 of `05-data-structures-part2.Rmd`, is that right? I.e., omit "basic subsetting"?

### CODE REVIEW

File: 04-data-structures-part1.Rmd [hunk 1]
```diff
@@ -178,16 +181,16 @@ Vectors can be any data type that we've already learned about. Let's make a
 character vector:
 
 ```{r}
-my_other_vector <- c("Harry", "Ron", "Hermione", "Ginny")
-my_other_vector
+hogwarts_students <- c("Harry", "Ron", "Hermione", "Ginny")
```
Comment (boshek) [inline 04-data-structures-part1.Rmd:184]: This is critical :)

File: 05-data-structures-part2.Rmd [hunk 2]
```diff
@@ -60,9 +60,6 @@ data frame in the environment pane.
 
 Note that this is a read-only view, it's not editable (which is a good thing!)
 
-
-For an object containing the data type `factor`, each different value represents what is called a `level`. In our case, the `factor` "coat" has 3 levels: "black", "calico", and "tabby".
```
Comment (boshek) [inline 05-data-structures-part2.Rmd:64]: This is good to remove too since R 4.0.0 will fix this.

**Comment by ateucher** [comment]
I think the plan is to stick to `read_csv`, but as Sam says as of R4.0 it will be `stringsAsFactors = FALSE` by default...

For lists and matrices, I think I will keep them in for completeness, but maybe not teach them? Ditto for basic data frame subsetting - keep it in but not teach it.

**Comment by boshek** [comment]
I think we can likely merge this?

**Comment by ateucher** [comment]
Yup I think so

**Pull request merged**